tests/run-make/split-debuginfo/Makefile

```diff
@@ -141,9 +205,12 @@ packed-remapped-single:
 # - `.dwp` present
 packed-remapped-scope:
```

Comment:
We should add a test with scope `split-debuginfo,split-debuginfo-path` that greps the entire `.dwp` file for remapped paths (i.e. without going through objdump or readelf).